velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -141,8 +152,8 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
   auto endBoundMinIdx = std::max(0, static_cast<int>(startBoundIndex) - 1);
   auto endBoundIndex = boost::random::uniform_int_distribution<uint32_t>(
       endBoundMinIdx, endBoundOptions.size() - 1)(rng_);
-  auto frameStart = frameBound(startBoundOptions[startBoundIndex]);
-  auto frameEnd = frameBound(endBoundOptions[endBoundIndex]);
+  auto frameStart = frameBound(startBoundOptions[startBoundIndex], "k0");
```

Comment:
Do you really need to pass column name as a parameter. It can be generated in place in frameBound lambda.

Reply:
+1.

Reply:
The frameStart is called with k0 and the second frameEnd it is called with k1 so they are two different string values.
Would you rather it be generated in the function with something like:
```
  int count = 0;

  auto frameBound =
      [this, minKValue, maxKValue, &argNames, &argTypes](
          core::WindowNode::BoundType boundType) -> const std::string {
    auto kValue = boost::random::uniform_int_distribution<int64_t>(
        minKValue, maxKValue)(rng_);


   auto colVal = k+std::to_string(count++);
```
This way the first time it will be k0 and the second time it will be k1
Or is there a cleaner way that it can be generated?

Reply:
@jkhaliqi : I simply meant the following 

```
auto frameBound = [this, minKValue, maxKValue, &argNames, &argTypes](
                        core::WindowNode::BoundType boundType) -> const std::string {
    auto kValue = boost::random::uniform_int_distribution<int64_t>(
        minKValue, maxKValue)(rng_);
    switch (boundType) {
      case core::WindowNode::BoundType::kUnboundedPreceding:
        return "UNBOUNDED PRECEDING";
      case core::WindowNode::BoundType::kPreceding:
        if (vectorFuzzer_.coinToss(0.5)) {
          // Generating column bounded k PRECEDING frames 50% of the time and
          // constant bounded the other 50%
          argTypes.push_back(INTEGER());
          argNames.push_back("k0");
          return fmt::format("k0 PRECEDING");
        }
        return fmt::format("{} PRECEDING", kValue);
      case core::WindowNode::BoundType::kCurrentRow:
        return "CURRENT ROW";
      case core::WindowNode::BoundType::kFollowing:
        if (vectorFuzzer_.coinToss(0.5)) {
          // Generating column bounded k FOLLOWING frames 50% of the time and
          // constant bounded the other 50%
          argTypes.push_back(INTEGER());
          argNames.push_back("k1");
          return fmt::format("k1 FOLLOWING");
        }
}
```

Reply:
Thanks for clarifying, updated!